pkg/reconciler/events/storage/storage.go

```diff
@@ -255,5 +256,5 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, storage *v1alpha1.CloudSt
 	}
 
 	// ok to remove finalizer.
-	return nil
+	return reconciler.NewEvent(corev1.EventTypeNormal, deleteNotificationSuccess, "Successfully deleted CloudStorageSource notification: %s", storage.Status.NotificationID)
```

Comment:
I'd rather leave this one as before, just return nil... The object will be deleted after this, so IMO no need to emit an deleteNotificationSuccess... We do emit, when there was a failure deleting, so that users can just describe the object and see what happened...
I think if we care about deletion events, we can use the ApiServerSource, listening on `CloudStorageSource` objects delete, and that should suffice... What do you think?

Reply:
done